integration/multi_server_forwarding_pipeline_test.go

```diff
@@ -193,21 +193,23 @@ func TestMultiServerForwardingPipeline(t *testing.T) {
 		leaders    = make(map[int]struct{})
 		leaderCh   = make(chan int, len(servers)/2)
 		numLeaders int32
+		wg         sync.WaitGroup
 	)
+	wg.Add(len(servers) / 2)
```

Comment:
Why divide by two here instead of using `len(servers)` and calling `wg.Done()` at the end of each goroutine?

Reply:
Because only half will become leader, and waiting for the servers to realize that takes 30s, so I'd rather not wait that long.